### ISSUE TITLE
high avg loss

### ISSUE THREAD

**Issue by mosca-ops** [AlexeyAB/darknet#7137]
the avg loss doesn't decrease, why? 
I'm using yolov2 tiny

this is my cfg:
> [net]
> batch=64
> subdivisions=8
> width=416
> height=416
> channels=3
> momentum=0.9
> decay=0.0005
> angle=0
> saturation = 1.5
> exposure = 1.5
> hue=.1
> 
> learning_rate=0.001
> burn_in=1000
> max_batches = 500200
> policy=steps
> steps=400000,450000
> scales=.1,.1
> 
> [convolutional]
> batch_normalize=1
> filters=16
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=32
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=256
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=512
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=1
> 
> [convolutional]
> batch_normalize=1
> filters=1024
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> ###########
> 
> [convolutional]
> batch_normalize=1
> size=3
> stride=1
> pad=1
> filters=512
> activation=leaky
> 
> [convolutional]
> size=1
> stride=1
> pad=1
> filters=40
> activation=linear
> 
> [region]
> anchors =  0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828
> bias_match=1
> classes=3
> coords=4
> num=5
> softmax=1
> jitter=.2
> rescore=0
> 
> object_scale=5
> noobject_scale=1
> class_scale=1
> coord_scale=1
> 
> absolute=1
> thresh = .6
> random=1

my dataset is on 600 images with 3 classes
the results of detector are:
> ![image](https://user-images.githubusercontent.com/62932720/102329097-3c9daa80-3f88-11eb-9208-9268c40e0ff1.png)
(this is on 6000 steps)

and the avg loss doesn't decrease under 0.19. I want to have avg loss under 0.05 because I also trained a model with the same cfg but with a dataset on 150 images and the avg loss was under 0.04. The result of detector on this dataset was:

> ![image](https://user-images.githubusercontent.com/62932720/102329778-14627b80-3f89-11eb-878c-4f27b0a858b3.png)

(this is on 7000 steps)

> ![image](https://user-images.githubusercontent.com/62932720/102329916-39ef8500-3f89-11eb-91c0-24c5cf028096.png)

(this is on 6000 steps)

later I want to do on a dataset with 3000 images. what am I doing wrong? Do I have to change something when increasing the dataset?
what can I do?
thanks
